include/z64view.h

```diff
@@ -55,4 +55,33 @@ typedef struct View {
 #define VIEW_ERROR_CHECK_EYE_POS(x, y, z) (void)0
 #endif
 
+View* View_New(struct GraphicsContext* gfxCtx);
```

Comment:
what are thoughts on `SCREEN_WIDTH` and `SCREEN_HEIGHT` going in here?
My first thought was a gfx related header cause thats mostly the contexts it makes sense in, but its also used for "is thing on screen" checks that arent necessarily graphics are drawing related

Reply:
was expecting a bit more conversation on this, do you agree with the change?
its the best I can think, of but dont think its perfect

Reply:
I do agree

Reply:
I don't agree with it. View is a higher level system for creating a 3D viewport using the RCP, whereas `SCREEN_WIDTH`/`SCREEN_HEIGHT` is used in lower-level systems like main.c and fault.c/fault_drawer that draw directly to the framebuffer via CPU. It doesn't make sense to make these systems dependent on view.h

Also note that moving `SCREEN_WIDTH`/`SCREEN_HEIGHT` now makes some scenes dependent on view.h, since ZAPD currently (incorrectly) extracts prerendered jpeg backgrounds like so: 

```c
u64 hut_room_0Background_003DA0[SCREEN_WIDTH * SCREEN_HEIGHT / 4] = {
#include "assets/scenes/indoors/hut/hut_room_0Background_003DA0.jpg.inc.c"
};```

Reply:
Yeah that makes sense. Any ideas on where it can go then? are we about to make its own header with those two things in it? lol

Anyway that conversation can happen outside of this PR, I agree with undoing this

Reply:
main.h seems like a decent place to put for now it since it already has gScreenWidth/gScreenHeight there.

Reply:
including main.h in other places in the codebase would feel pretty weird ngl lol

Reply:
reverted to placed in z64.h for now

Reply:
thanks, sorry for the flip flopping on this

Reply:
no problem!